wordpress-mobile/WordPress-iOS
# Lock Screen Widgets: Add more configurations

Fixes #21494 

Added:
- [Add All Time Views Lock Screen Widget](https://github.com/wordpress-mobile/WordPress-iOS/commit/0f678ea1a22db8477d52a593289bec5c8a414ea9)
- [Add Today Views and Visitors Lock Screen Widget](https://github.com/wordpress-mobile/WordPress-iOS/commit/d4e9786e3ddef261776912f8c61b6ec5b7c97729)
- [Add All Time Views and Visitors Lock Screen Widget](https://github.com/wordpress-mobile/WordPress-iOS/commit/4800c8c99afbdee05c399363c5dd87b13ada22bb)
- [Add All Time Posts and Best Views Stat Widget](https://github.com/wordpress-mobile/WordPress-iOS/commit/413e1fda559e1ba5467923a27827009c22cd4d49)

## To test

1. Enable lockScreenWidget feature flag in `FeatureFlag.swift`
2. Open Jetpack
3. Log into account
4. Go to lock screen, long press, customize, tap on widget area, select Jetpack app
5. Scroll through widgets
6. Confirm new widget configuration exists
7. Confirm their numbers match Stats
8. Confirm tapping on Today widget opens "Day" view
9. Confirm tapping on All time widget opens "Insights"view

## Regression Notes
1. Potential unintended areas of impact

None

2. What I did to test those areas of impact (or what existing automated tests I relied on)

None

3. What automated tests I added (or what prevented me from doing so)

None

PR submission checklist:


UI Changes testing checklist:

## Video

https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/454b4289-94eb-4e47-8558-463ac01e0523
